go.mod

```diff
@@ -18,13 +21,11 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.19.1-0.20210203200406-65673ad3657b
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver v0.19.1-0.20210203200406-65673ad3657b
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/collectdreceiver v0.19.1-0.20210203200406-65673ad3657b
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/dockerstatsreceiver v0.19.1-0.20210203200406-65673ad3657b
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver v0.19.1-0.20210203200406-65673ad3657b
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver v0.19.1-0.20210203200406-65673ad3657b
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kubeletstatsreceiver v0.19.1-0.20210203200406-65673ad3657b
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusexecreceiver v0.19.1-0.20210203200406-65673ad3657b
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/receivercreator v0.19.1-0.20210203200406-65673ad3657b
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.19.1-0.20210203200406-65673ad3657b
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.19.1-0.20210203200406-65673ad3657b // indirect
```

Comment:
Wait how did this get added back?

Reply:
Oh was removed from https://github.com/signalfx/splunk-otel-collector/pull/84/files but not go.mod. Hopefully can be removed?

Reply:
kubletstats requires redis

```
go: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kubeletstatsreceiver@v0.19.1-0.20210203200406-65673ad3657b requires
	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver@v0.0.0-00010101000000-000000000000
```